src/Symfony/Component/HttpClient/Response/CurlResponse.php

```diff
@@ -150,8 +148,6 @@ public function __construct(CurlClientState $multi, $ch, array $options = null,
     public function getInfo(string $type = null)
     {
         if (!$info = $this->finalInfo) {
-            self::perform($this->multi);
```

Comment:
Performing here totally kills performances when processing a large number of requests.